cypress.config.mjs

```diff
@@ -1,10 +1,31 @@
 import { defineConfig } from 'cypress';
+import { getProducts } from './tools/gulptasks/lib/test.js';
+
+const products = new Set();
+
+for (const product of getProducts()) {
+    switch (product) {
+        case 'Dashboards':
```

Comment:
I guess we don't need to distinguish here between the products because later on you overwrite the `specPattern` in the `test/cypress/dashboards/config.mjs` file.

Reply:
you are right! you can ignore it 😉